src/components/QuickOpenModal.js

```diff
@@ -121,33 +127,59 @@ export class QuickOpenModal extends Component<Props, State> {
     this.setState({ results });
   };
 
+  generalQuery = (query: string) => {
```

Comment:
Note: I don't know if the name `generalQuery` makes sense. It took me a while to figure out that you meant the `?` as the "general" query. I would have thought of the default (sources) as the "general" in this context. I'd maybe go with a name like `shortcutQuery` or `hintQuery` to more accurately convey what details are shown.